lib/phut/cli.rb

```diff
@@ -0,0 +1,55 @@
+require 'rake'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.